unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/FavoritesSubSection/FavoritesesSubSectionComponentController.cs

```diff
@@ -83,29 +82,51 @@ public void RequestAllFromAPI()
             OnCompleted: OnRequestedEventsUpdated);
     }
 
-    private void OnRequestedEventsUpdated(List<HotSceneInfo> placeList)
+    private void OnRequestedEventsUpdated(List<PlaceInfo> placeList)
     {
         friendsTrackerController.RemoveAllHandlers();
 
         favoritesFromAPI = placeList;
 
-        view.SetFavorites(PlacesAndEventsCardsFactory.CreatePlacesCards((TakeAllForAvailableSlots(placeList))));
+        view.SetFavorites(ConvertPlacesToModels(TakeAllForAvailableSlots(placeList)));
         view.SetShowMoreFavoritesButtonActive(availableUISlots < favoritesFromAPI.Count);
     }
 
-    internal List<HotSceneInfo> TakeAllForAvailableSlots(List<HotSceneInfo> modelsFromAPI) =>
+    private List<PlaceCardComponentModel> ConvertPlacesToModels(List<PlaceInfo> placeInfo)
+    {
+        List<PlaceCardComponentModel> modelsList = new List<PlaceCardComponentModel>();
+
+        foreach (var place in placeInfo)
+        {
+            modelsList.Add(
+                new PlaceCardComponentModel()
+                {
+                    placePictureUri = place.image,
+                    placeName = place.title,
+                    placeDescription = place.description,
+                    placeAuthor = place.contact_name,
+                    numberOfUsers = place.user_count,
+                    coords = new Vector2Int(),
+                    parcels = new Vector2Int[1]
+                });
+        }
+
+        return modelsList;
+    }
+
+    internal List<PlaceInfo> TakeAllForAvailableSlots(List<PlaceInfo> modelsFromAPI) =>
         modelsFromAPI.Take(availableUISlots).ToList();
```

Comment:
How often is this called? LinQ generates lot of allocations, it's better to do it manually.